server/src/main/java/org/apache/cassandra/sidecar/server/MainModule.java

```diff
@@ -476,14 +476,14 @@ public DriverUtils driverUtils()
 
     @Provides
     @Singleton
-    public InstancesConfig instancesConfig(Vertx vertx,
-                                           SidecarConfiguration configuration,
-                                           CassandraVersionProvider cassandraVersionProvider,
-                                           SidecarVersionProvider sidecarVersionProvider,
-                                           DnsResolver dnsResolver,
-                                           CQLSessionProvider cqlSessionProvider,
-                                           DriverUtils driverUtils,
-                                           MetricRegistryFactory registryProvider)
+    public InstancesMetadata instancesConfig(Vertx vertx,
```

Comment:
rename the method as well
```suggestion
    public InstancesMetadata instancesMetadata(Vertx vertx,
```